packages/strapi-design-system/src/SimpleNav/SimpleNav.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Flex } from '../Flex';
+
+export const SimpleNav = (props) => {
+  return <Flex {...props} />;
```

Comment:
Semantically I believe this should be a `nav` item then wrapping `ul` element because its a nav list.